crowdsec-docs/unversioned/troubleshooting/issue_lp_offline.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Log Processor Offline
-id: issue_log_processor_offline
+id: issue_lp_offline
```

Comment:
Look for errors -> too specific errors, jsut tell the user to look at the output
Also verify the API endpoint in /etc/crowdsec/config.yaml -> kill, too specific
Test network connectivity: -> kill, too specific
If the Local API service is unavailable + If the central LAPI is unreachable from the agent -> doublon ?